src/arrow-function/index.js

```diff
@@ -12,10 +12,14 @@ export function add(){
 
   var sum = 0;
   for (var i = 0; i < arguments.length; i++) {
-    if(arguments[i])
-      sum += arguments[i];
+    if(isNumber(arguments[i]))
+      sum += +arguments[i];
     else
       throw new Error('You can only sum numbers');
   }
   return sum;
 }
+
+function isNumber(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
Hey Pablo, thanks for your PR. Here are few points to take in account:
- ```'2'``` should not be converted to ```2```
- ```isFinite``` will parse the value by default so you might want to use ```Number.isFinite``` (ES6) instead